TTS/bin/train_glow_tts.py

```diff
@@ -467,7 +467,7 @@ def evaluate(data_loader, model, criterion, ap, global_step, epoch):
 
 def main(args):  # pylint: disable=redefined-outer-name
     # pylint: disable=global-variable-undefined
-    global meta_data_train, meta_data_eval, symbols, phonemes, speaker_mapping
+    global meta_data_train, meta_data_eval, symbols, phonemes, model_characters, speaker_mapping
```

Comment:
For readable might be nice to have not so many global variables for readability. Could be put into a [dataclass](https://docs.python.org/3/library/dataclasses.html) that is shared between all train scripts. But this is probably a different PR.

Reply:
I was not even aware of the dataclass. It makes sense and it is definitely a better way to handle it.